docs/related.md

```diff
@@ -15,6 +15,11 @@ or [pull request](https://github.com/kubernetes/helm/pulls).
 - [The Workflow "Umbrella" Helm Chart](https://deis.com/blog/2017/workflow-chart-assembly)
 - [GitLab, Consumer Driven Contracts, Helm and Kubernetes](https://medium.com/@enxebre/gitlab-consumer-driven-contracts-helm-and-kubernetes-b7235a60a1cb#.xwp1y4tgi)
 
+## Videos
+
+- [KubeCon2016: Delivering Kubernetes-Native Applications by Michelle Noorali](https://www.youtube.com/watch?v=zBc1goRfk3k&index=49&list=PLj6h78yzYM2PqgIGU1Qmi8nY7dqn9PCr4)
+- [KubeCon2016: Ancestry's Journey to Kubernetes by Paul MacKay, Ancestry.com](https://www.youtube.com/watch?v=zLn_H5jaWxQ&index=69&list=PLj6h78yzYM2PqgIGU1Qmi8nY7dqn9PCr4)
```

Comment:
hi @michelleN, Thank you for your comment. I have searched for the word `Taking the Helm` and listed it. Certainly, I think that the second subject is not Helm tool. I'll fix it :octocat: